cgmanifests/submodules/cgmanifest.json

```diff
@@ -219,22 +219,12 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "80d452484c5409444b0ec19383faa84bb7a4d351",
+          "commitHash": "59a2ac2745d8a57ac94c6accced73620d59fb844",
```

Comment:
this doesn't seem to be the right commit.
we seem to be pointing to 2.6.1  , the commit here is for 2.6.0

Reply:
It is correct. See https://github.com/onnx/onnx/tree/master/third_party
"pybind11 @ 59a2ac2"

Reply:
We should have another entry in https://github.com/microsoft/onnxruntime/blob/master/cgmanifests/cgmanifest.json for 2.6.1.

Reply:
that's onnx repo. i think you meant to look at https://github.com/microsoft/onnxruntime/blob/master/cmake/external/pybind11.cmake#L11

Reply:
or should we have ORT stay in sync with ONNX for pybind11?

Reply:
I feel it is ok to have a different version in ORT.